pdns/qtype.hh

```diff
@@ -125,7 +125,8 @@ public:
     CAA=257,
     DLV=32769,
     ADDR=65400,
-    ALIAS=65401
+    ALIAS=65401,
+    XPF=65422
```

Comment:
Remove this?

Reply:
Hmm, I guess that would break `mastermake`. Now I see why fixing `sdig` is harder.